src/mobu/models/business/sia.py

```diff
@@ -0,0 +1,54 @@
+"""Base models for SIA-related monkey business."""
+
+from __future__ import annotations
+
+from astropy.time import Time
+from pydantic import BaseModel, Field
+
+from .base import BusinessData, BusinessOptions
+
+__all__ = ["SIA2SearchParameters", "SIABusinessData", "SIABusinessOptions"]
+
+
+class SIA2SearchParameters(BaseModel):
+    """The parameters of an SIAv2 query."""
+
+    ra: float
+    dec: float
+    radius: float
+    time: list[float]
+
+    @property
+    def pos(self) -> tuple[float, float, float]:
+        return self.ra, self.dec, self.radius
+
+    def to_pyvo_sia2_params(self) -> dict:
+        """Return the query as a dictionary in a form that
+        pyvo's SIA2 search expects it. We transform the time strings to
+        astropy Time objects and then to datetime.
+
+        Returns
+        -------
+            dict: The query as a dictionary.
+        """
+        times = [Time(str(t), format="mjd").to_datetime() for t in self.time]
+        return {"pos": self.pos, "time": times}
+
+    def __str__(self) -> str:
+        """Return a string representation of the query."""
+        times = ", ".join([str(t) for t in self.time])
+        return f"SIA2Query(pos={self.pos}, time=[{times}])"
```

Comment:
If I saw this, I would assume from the formatting that this is a repr of some class somewhere, but it's not. What do you think of something like `SIAv2 parameters: .....`?